difflint/lint.py

```diff
@@ -16,8 +16,7 @@
 
 @functools.lru_cache()
 def _get_enabled_linters_config_path():
-    """
-    Returns the path of the file describing which linters are enabled
+    """Returns the path of the file describing which linters are enabled
```

Comment:
Yeah, I screwed up and `commit --amend`'ed them.  I need to remember the best way to correct this.